src/usdm-client.ts

```diff
@@ -153,7 +154,9 @@ export class USDMClient extends BaseRestClient {
     return this.get('fapi/v1/markPriceKlines', params);
   }
 
-  getMarkPrice(params?: Partial<BasicSymbolParam>): Promise<any> {
+  getMarkPrice(
+    params?: Partial<BasicSymbolParam>
+  ): Promise<MarkPrice | MarkPrice[]> {
```

Comment:
By the way, if you wanted to get a smart return type, I recently discovered this is one way of getting the correct return type (depending on whether the symbol param is included:
https://github.com/tiagosiebler/bybit-api/blob/master/src/spot-client.ts#L89-L95

Doesn't have to be in this PR, but it's a nice touch